app/driver.go

```diff
@@ -221,6 +239,11 @@ func deployCSIDriver(kubeClient *clientset.Clientset, lhClient *lhclientset.Clie
 		return err
 	}
 
+	snapshotSupportEnabled, err := util.IsKubernetesVersionAtLeast(kubeClient, types.CSISnapshotterMinVersion)
```

Comment:
We need to think about what's our expectation for the snapshotter controller. I would prefer us to deploy it if possible, though strictly speaking it's not a Longhorn component.

Reply:
Agree, but that can be a separate PR, this way we can test the csi snapshotter interactions already.